examples/step-89/doc/intro.dox

```diff
@@ -0,0 +1,221 @@
+same position. In this situation it is hardly possible to construct a conforming mesh in every
+time step.
+
+Besides this, non-matching methods can be extended to Chimera methods (elements overlap).
+Chimera methods can help to reduce the pressure on mesh generation tools since different regions
+of a mesh (that may overlap) can be considered independently.
+
+Different methods exist to treat non-matching interfaces. Within this tutorial, we concentrate on
+two methods: Point-to-point interpolation and Nitsche-type mortaring.
+
+<h4>%Point-to-point interpolation</h4>
+
+@image html step-89-p2p.svg "" width=25%
+
+Point-to-point interpolation is a naive approach. The points in which values/gradients are
+queried in the coupling terms are defined by the quadrature points on the element face of
+element $K^-$. As it can be seen from the picture this approach might be subject to aliasing
+in some cases. In the picture, information from element $K_1^+$ is completely neglected.
+
+<h4>%Nitsche-type mortaring</h4>
+
+@image html step-89-mortar.svg "" width=25%
+
+Mortaring is the process of computing intersections and is not related to the Mortar method which
+enforces the coupling via Lagrange multipliers. Obtained intersections are also referred to as
+mortars. On each mortar a new integration rule is defined. The integral of the face of element
+$K^-$ is computed on the intersections. This way, the numerical integration is exact as long
+as a sufficient number of integration points is used.
+In this tutorial, the intersections are computed using `CGAL`. Therefore, `deal.II` has
+to be configured with `DEAL_II_WITH_CGAL` for the Nitsche-type mortaring implementation.
+
+<h3>%FERemoteEvaluation</h3>
+
+FERemoteEvaluation is a wrapper class which provides a similar interface to, e.g., FEEvaluation to
+access values over non-matching interfaces in matrix-free loops. A detailed description on how to setup
+the class and how to use it in actual code is given below on hands-on examples. Within this tutorial we only
+show the usage for non-matching discretizations. Note however, that FERemoteEvaluation can also be used in
+other settings such as volume coupling. Under the hood, Utilities::MPI::RemotePointEvaluation is used to query
+the solution or gradients at quadrature points. A detailed description how this is done can be found in step-87.
+The main difference in the usage of FERemoteEvaluation compared to FEEvaluation is that the interpolated
+values/gradients of the finite element solution at all the quadrature points are precomputed globally <i>before</i>
+the loop that passes through the cells/faces of the mesh (i.e., near the place where the communication takes place)
+instead of performing the interpolation out of the vector on a cell-by-cell basis.
+
+I.e., the code differs as follows: On the one hand, the standard code to evaluate fluxes via FEEvaluation
+@code
```

Comment:
I would mention again here the difference between A and B instead of just saying the code differs. It's a bit confusing for the reader at this point the difference is between what and what.

Reply:
The difference is one line above. I simply rephrased the sentence :)

Reply:
Perfect :)